src/Containers/Project/Project.js

```diff
@@ -8,17 +11,44 @@ export class Project extends Component{
       expanded: false
     }
   }
+  
```

Comment:
Great checks for expanding vs. closing the currentExpanded project as well as check for if we already have palettes for the project